PyInstaller/building/build_main.py

```diff
@@ -558,12 +558,9 @@ def _check_python_library(self, binaries):
             binaries.append((os.path.basename(python_lib), python_lib, 'BINARY'))
             logger.info('Using Python library %s', python_lib)
         else:
-            msg = """Python library not found! This usually happens on Debian/Ubuntu
-where you need to install Python library:
-
-  apt-get install python3-dev
-  apt-get install python-dev
```

Comment:
@yyuu Please keep also the ubuntu/debian specific message. And recommend trying installing package `libpythonX.Y`. In the past we had many complains and that is the reason why to recommend installing libpython on Debian/Ubuntu. Thanks.